pkg/route/validation/validation.go

```diff
@@ -90,6 +101,35 @@ func validateRoute(route *routev1.Route, checkHostname bool) field.ErrorList {
 		result = append(result, err)
 	}
 
+	if route.Spec.HTTPHeaders != nil {
+		if len(route.Spec.HTTPHeaders.Actions.Response) != 0 || len(route.Spec.HTTPHeaders.Actions.Request) != 0 {
+			if route.Spec.TLS != nil && route.Spec.TLS.Termination == routev1.TLSTerminationPassthrough {
+				specTLS := field.NewPath(specTLSTerminationString)
+				result = append(result, field.NotSupported(specTLS, route.Spec.TLS.Termination, []string{"Only edge and re-encrypt routes are supported for providing customized headers."}))
```

Comment:
This is a different error message than what appear in openshift/api CEL: https://github.com/openshift/api/pull/1364/files#diff-4dfca44f3f68bff8b6381ff3fac676c6be96da9ec1d15224bbd70f839744eb40R50 

"header actions are not permitted when tls termination is passthrough."

Shouldn't they be the same?

Reply:
I think as far as the user is understanding the issue then that should be fine.

Reply:
CEL's are still present for route.